st2common/st2common/util/spec_loader.py

```diff
@@ -77,7 +77,7 @@ def load_spec(module_name, spec_file, allow_duplicate_keys=False):
 
     # 1. Check for duplicate keys
     if not allow_duplicate_keys:
-        yaml.load(spec_string, UniqueKeyLoader)
```

Comment:
This function is only used to load ``openapi.yaml.j2`` file which is controlled by us (developers), which means there should be no chance for "yaml type injection", but I do agree and still think it's better and safer to use ``yaml.safe_load``.